src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+const getVisibleMovies = (queryToFilter: string) => (
+  moviesFromServer.filter(movie => (
+    movie.title.toLowerCase().includes(queryToFilter.toLowerCase())
+    || movie.description.toLowerCase().includes(queryToFilter.toLowerCase())
+  ))
+);
```

Comment:
Better create constants f.e. `descriptionNormalized` and use it. Additional lines of code but a huge improvement in readability.